### PR TITLE
Interface added

### PR DESCRIPTION
**Pull request by Snafkin547**

### CODE REVIEW

File: packages/server/src/email/email.service.ts [hunk 1]
```diff
@@ -5,22 +5,15 @@ import * as ses from 'node-ses';
 @Injectable()
 export class EmailService {
   private readonly client = ses.createClient({} as any);
-
+
```
Comment (cbolles) [inline packages/server/src/email/email.service.ts]: NIT (short for nit pick): Best to not have white space changes. I've added a feature to my IDE to remove extra spaces/etc at the end of lines for things like that.

File: packages/server/src/email/email.service.ts [hunk 2]
```diff
@@ -5,22 +5,15 @@ import * as ses from 'node-ses';
...
-        from: 'test@email.sail.codes',
-        subject: 'Greetings',
-        message: 'Hello',
-        altText: 'plain text',
-        to: 'hishii@bu.edu',
-      },
+        msg,
       function (err, data, res) {
-        console.log(res);
+        // console.log(res);
```
Comment (cbolles) [inline packages/server/src/email/email.service.ts]: NIT: Want to remove commented out code unless it has a use later on

**Review by cbolles — COMMENTED** [review]
Have some stylistic recommendations

File: packages/server/src/email/email.service.ts [hunk 3]
```diff
@@ -5,17 +5,14 @@ import * as ses from 'node-ses';
...
     const msg: EmailMessage = JSON.parse(message.Body) as EmailMessage;
     // Give SES the details and let it construct the message for you.
-    this.client.sendEmail(
-        msg,
-      function (err, data, res) {
-        // console.log(res);
-      },
-    );
+    this.client.sendEmail(msg, function (err, data, res) {
+      // console.log(res);
```
Comment (cbolles) [inline packages/server/src/email/email.service.ts]: NIT: Last commented out line

**Review by cbolles — CHANGES_REQUESTED** [review]
Other then the warnings that came from GitHub looks good except for a left over commented out line

**Review by gregfrasco — APPROVED** [review]

**Review by cbolles — APPROVED** [review]

**Pull request merged**